File: SGA_Desktop/SGA_Desktop/Models/PaletDto.cs
using System;

namespace SGA_Desktop.Models
{
	public class PaletDto
	{
		public Guid Id { get; set; }
		public short CodigoEmpresa { get; set; }
		public string Codigo { get; set; } = "";
		public string Estado { get; set; } = "";
		public string? TipoPaletCodigo { get; set; }
		public DateTime FechaApertura { get; set; }
		public DateTime? FechaCierre { get; set; }
		public int UsuarioAperturaId { get; set; }
		public int? UsuarioCierreId { get; set; }
		public decimal? Altura { get; set; }
		public decimal? Peso { get; set; }
		public bool EtiquetaGenerada { get; set; }
		public bool IsVaciado { get; set; }
		public DateTime? FechaVaciado { get; set; }
	}
}

File: SGA_Android/app/src/main/java/com/example/sga/data/dto/traspasos/PaletDto.kt
package com.example.sga.data.dto.traspasos

import com.google.gson.annotations.SerializedName

data class PaletDto(
    val id: String,

    @SerializedName("codigo")
    val codigoPalet: String,

    @SerializedName("tipoPaletCodigo")
    val tipoPaletCodigo: String,

    @SerializedName("estado")
    val estado: String,

    @SerializedName("fechaApertura")
    val fechaApertura: String?,

    @SerializedName("fechaCierre")
    val fechaCierre: String?,

    @SerializedName("usuarioAperturaId")
    val usuarioAperturaId: Int?,

    @SerializedName("usuarioCierreId")
    val usuarioCierreId: Int?,

    @SerializedName("ordenTrabajoId")
    val ordenTrabajoId: String?,

    @SerializedName("altura")
    val altura: Double?,

    @SerializedName("peso")
    val peso: Double?,

    @SerializedName("etiquetaGenerada")
    val etiquetaGenerada: Boolean,

    @SerializedName("isVaciado")
    val isVaciado: Boolean,

    @SerializedName("fechaVaciado")
    val fechaVaciado: String?,

    @SerializedName("codigoEmpresa")
    val codigoEmpresa: Short,

    @SerializedName("codigoGS1")
    val codigoGS1: String?

)
